lib/releases.js

```diff
@@ -57,7 +58,9 @@ const findReleases = async ({
     ? commitishFilteredReleases.filter((r) => r.tag_name.startsWith(tagPrefix))
     : commitishFilteredReleases
   const sortedPublishedReleases = sortReleases(
-    filteredReleases.filter((r) => !r.draft && !r.prerelease)
+    filteredReleases.filter(
+      (r) => !r.draft && (!excludePreReleases || !r.prerelease)
```

Comment:
Logic checks out, but it doesn't read right.
```
!r.draft && (!r.prerelease || includePreReleases)
```
would be preferable.

I'm quite concerned about the amount of boolean logic in this function already which is why I'm even giving this a comment. It probably needs to get reflowed entirely, but that's out of scope here.

Reply:
I've changed the configuration to `include-pre-releases` which makes more sense :) Thanks for your input!